gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberSettings.java

```diff
@@ -336,4 +201,212 @@ public Object extractNextToken(ListSubscriptionsResponse payload) {
               return payload.getSubscriptionsList();
             }
           };
+
+  public static class Builder extends ServiceApiSettings.Builder {
+    private final ImmutableList<ApiCallSettings.Builder> methodSettingsBuilders;
+
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'createSubscriptionSettings' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2381705098/issues/source?bid=3123109&fileBranchId=3225976#l208)